VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -174,7 +174,7 @@
                           android:id="@+id/course_unit_nav_bar"
                           android:measureWithLargestChild="true"
                           android:layout_width="match_parent"
-                          android:layout_height="wrap_content"
+                          android:layout_height="55dp"
```

Comment:
We should define some sort of footer heigh constant instead of hardcoding it here.